query-node/substrate-query-framework/index-builder/src/QueryEvent.ts

```diff
@@ -5,7 +5,16 @@ interface EventParameters {
   [key: string]: Codec;
 }
 
-export default class QueryEvent {
+export interface JoyEvent {
```

Comment:
Since `index-builder` package is for substrate chains, could be better if we rename it